src/mpid/ch4/generic/am/mpidig_am_part_callbacks.c

```diff
@@ -125,6 +125,7 @@ int MPIDIG_part_cts_target_msg_cb(int handler_id, void *am_hdr, void *data,
 
     MPIDIG_PART_REQUEST(part_sreq, peer_req_ptr) = msg_hdr->rreq_ptr;
     MPIDIG_PART_REQ_INC_FETCH_STATUS(part_sreq);
+    mpi_errno = MPIDIG_post_pready(part_sreq, is_local);
```

Comment:
Might be better to update the function comment of `MPIDIG_part_cts_target_msg_cb` as well.

Reply:
Done.